cypress/integration/nav_bar_logout_spec.js

```diff
@@ -1,21 +1,16 @@
+const signUp = require('../support/signup_helper')
+const signIn = require('../support/signin_helper')
+
```

Comment:
Is this require necessary now because they've now been placed in a /support/ folder?

Reply:
I tried without and it didn't work